packages/RNTester/e2e/__tests__/Modal-test.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails oncall+react_native
+ * @format
+ */
+
+/* global device, element, by, expect */
+const {
+  openComponentWithLabel,
+  openExampleWithTitle,
+} = require('../e2e-helpers');
+
+describe('Modal', () => {
+  beforeAll(async () => {
+    await device.reloadReactNative();
+    await openComponentWithLabel(
+      '<Modal>',
+      '<Modal> Component for presenting modal views.',
+    );
+  });
+
+  it('Simple Modal should show content on button click', async () => {
```

Comment:
Hey @YashKumarVerma!

Jani has some [advice](https://github.com/MLH-Fellowship/react-native/pull/25#discussion_r440089871) on writing test cases name. You can check it and see if it can be improved.